src/tunnel/client/client.rs

```diff
@@ -133,6 +137,9 @@ impl WsClient {
         remote_addr: RemoteAddr,
         connector: impl TunnelConnector,
```

Comment:
The delay should be better configurable per tunnel/connection instead of putting the config globally for the whole client.

You can take as example the timeout parameter, https://github.com/erebe/wstunnel/blob/main/src/tunnel/mod.rs#L41  
It should be parsed from the tunnel command line (or use a default value) and passed as parameter in `run_reverse_tunnel`

Reply:
ack, will try that